config/game_options.txt

```diff
@@ -597,3 +597,14 @@ MODETIER_VOTING
 
 ## Number of modes dropped by the modetier vote during mode selection, after vote.
 DROPPED_MODES 3
+
+FLAVOR_OBJECTIVE_TRAITOR Teach the heads of staff a lesson they will never forget.=1
+FLAVOR_OBJECTIVE_TRAITOR Show Nanotrasen the utility of a pure oxygen atmosphere.=2
+FLAVOR_OBJECTIVE_TRAITOR Waffle Co. wants you! That’s right, YOU! To cause as much humorous terrorism against Nanotrasen as possible! How? We don’t care as long as it’s entertaining! Be as creative and exciting as possible when carrying out your dirty deeds. Have fun!=1
+FLAVOR_OBJECTIVE_TRAITOR Kill one of the station's beloved pets. Make a show of it, though you don't have to reveal yourself.=3
+FLAVOR_OBJECTIVE_TRAITOR Get illegal technology spread through the station.=3
+FLAVOR_OBJECTIVE_TRAITOR Slow down the process of research as much as possible." = (owner.assigned_role in list("Research Director", "Scientist", "Roboticist") ? 6 : 0),
+FLAVOR_OBJECTIVE_TRAITOR Channel your inner rat. Cut wires throughout the station." = (owner.assigned_role in list("Station Engineer", "Atmos Technician", "Assistant") ? 6 : 0),
```

Comment:
This will become cancerously annoying for engineers very very quickly.
Not only is it one of the least creative things to do, engineers now have even more additional busy-work.
If we take into account bad map design when it comes to wiring, you can very easily cut just a select few wires and depower most of the station.
If I encounter a traitor doing this as an engineer, I'm just going to stop trying to repair it or cryo. This **is not remotely fun in any way.** This objective isn't interesting, all it does is lower round quality by removing as much fun as possible, there's nothing this objective contributes to the round.

You have some quality objectives here but this one kinda ruins the rest of them.

Reply:
removed